src/pages/HomePage.jsx

```diff
@@ -3,17 +3,23 @@ import DefaultLayout from '../layouts/DefaultLayout';
 import TodoList from '../components/TodoList';
 import AddTodoForm from '../components/AddTodoForm';
 import EmptyListMessage from '../components/EmptyListMessage';
-
+import LoadingSpinner from '../components/LoadingSpinner';
 import useTodoManager from '../hooks/useTodoManager';
 
 function HomePage() {
-  const [todoList, addTodo, removeTodo] = useTodoManager();
+  const [todoList, addTodo, removeTodo, isLoading] = useTodoManager();
```

Comment:
I might be more descriptive in this name (`isTodoListLoading`) because if you end up w/ multiple elements on the page that load this will become confusing